hw3/02-url.js

```diff
@@ -1,6 +1,35 @@
 const http = require('http');
 
-const port = process.env.PORT || 5000;
+const port = process.env.PORT || 5001;
+
+const getURLData = (params) => { // Gets data from url and parses it
```

Comment:
I separated a lot of my functionality into separate functions like you did which I think is great and makes for better readability.